### PR TITLE
[Add] HelScans

### PR DESCRIPTION
**Pull request by beingsuz**
Add support for [HelScans](https://helscans.com/)

### CODE REVIEW

File: src/HelScans/HelScans.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+};
+
+export class HelScans extends MangaStream {
+	baseUrl: string = DOMAIN;
+
+	override configureSections(): void {
+		this.homescreen_sections["new_titles"].enabled = false;
+	}
+
+	override async getCloudflareBypassRequestAsync(): Promise<Request> {
```
Comment (TheNetsky) [inline src/HelScans/HelScans.ts]: Is this needed?

File: src/HelScans/HelScans.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { BadgeColor, ContentRating, SourceInfo, SourceIntents, Request } from "@paperback/types";
+
+import { getExportVersion, MangaStream } from "../MangaStream";
+
+const DOMAIN = "https://helscans.com";
+
+export const HelScansInfo: SourceInfo = {
+	version: getExportVersion("0.0.6"),
```
Comment (TheNetsky) [inline src/HelScans/HelScans.ts]: Set to 0.0.0

File: src/HelScans/HelScans.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	contentRating: ContentRating.MATURE,
+	websiteBaseURL: DOMAIN,
+	intents: SourceIntents.MANGA_CHAPTERS | SourceIntents.HOMEPAGE_SECTIONS | SourceIntents.CLOUDFLARE_BYPASS_REQUIRED | SourceIntents.SETTINGS_UI,
+	sourceTags: [],
+};
+
+export class HelScans extends MangaStream {
+	baseUrl: string = DOMAIN;
+
+	override configureSections(): void {
```
Comment (TheNetsky) [inline src/HelScans/HelScans.ts:31]: You're not following eslint, semicolons are not allowed.

**Review by TheNetsky — CHANGES_REQUESTED** [review]

**Comment by beingsuz** [comment]
Sorry gonna fix those tomorrow just did a quick copies of asura file and edited the domain

**Pull request merged**